requests/utils.py

```diff
@@ -224,11 +224,11 @@ def from_key_val_list(value):
     ::
 
         >>> from_key_val_list([('key', 'val')])
-        OrderedDict([('key', 'val')])
+        collections.OrderedDict([('key', 'val')])
```

Comment:
It looks like this is a cli example. We don't want the "collection" lead here. The interpreter is just showing the returned objects repr. Same with the line below this.